### DISCUSSION TITLE
How can I get binding instance to call UpdateSource() when using UpdateSourceTrigger=Explicit?

### DISCUSSION THREAD

**Discussion by laolarou726** [AvaloniaUI/Avalonia#15448] [Q&A]
Currently I wrote my binding expr in XAML:

```xaml
<Slider Value="{Binding UIManager.BgBlurRadius.Value, UpdateSourceTrigger=Explicit}"" />
```

How should I get the binding instance of the Value property to call the UpdateSource()

**Comment by maxkatz6** [comment]
It's not yet part of API: https://github.com/AvaloniaUI/Avalonia/issues/14965
